src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -341,6 +341,14 @@ public function reset_tracking_options() {
 	}
 
 	public function continue_import() {
+		global $wpdb;
+		if ( empty( $wpdb->get_var( "SELECT GET_LOCK('tribe_ea_csv_import',10)" ) ) ) {
```

Comment:
* :warning: Usage of a direct database call is discouraged.
* :warning: Direct database call without caching detected. Consider using wp_cache_get() / wp_cache_set() or wp_cache_delete().

<sub>hash: ce1b8cddea9c39ccb95cdb83923880fd</sub>